blog/20250108-introducing-geojupyter/index.md

```diff
@@ -0,0 +1,107 @@
+---
+title: "Introducing GeoJupyter"
+date: "2025-01-08"
+description: |
+  We believe that geospatial data analysis and exploration should be free, open source,
+  and accessible for everyone!
+author:
+  - name: "Matt Fisher"
+    orcid: "0000-0003-3260-5445"
+categories:
+  - "Announcements"
+---
+
+## :jigsaw: Need
+
+**We believe that geospatial data analysis and exploration should be free, open source,
+and accessible for everyone!**
+
+Currently, a divide exists between two worlds: The desktop GIS world, in which QGIS and
+ArcGIS provide intuitive visualization-first interfaces to exploring data; and the
+programming-based GIS world which provides cutting-edge editing, reproducibility, and
+automated publishing experiences.
+
+Between those two worlds, users of programming-based workflows face the daunting problem
+of a blank page and steep learning curves, while non-coders are missing out on the
+power of programming-based workflows to enable flexible alterations, version control,
+and low-friction reproducibility and publishing.
```

Comment:
I'd update to "low-friction modular reuse, reproducibility and publishing."

Reply:
>modular reuse

Ooo, love this!

Reply:
01f1612